build/mvn

```diff
@@ -70,9 +70,10 @@ install_app() {
 # Install maven under the build/ folder
 install_mvn() {
   local MVN_VERSION="3.3.9"
+  local APACHE_MIRROR=${APACHE_MIRROR:-https://archive.apache.org/dist}
```

Comment:
I didn't use a trailing slash in the definition of the default value here since I'd rather have 2 slashes at the end (if the user did define it) instead of no slash.